### ISSUE TITLE
Remove JUST_MY_CODE from debug options.

### ISSUE THREAD

**Issue by fabioz** [microsoft/debugpy#64]

**Comment by fabioz** [comment]
@karthiknadig should it show a deprecation message in case an `options` or `debugOptions` parameter is detected or can I just remove it altogether?

**Comment by fabioz** [comment]
I did the pull request but then I noted that `debugpy` tests pass `debugOptions`, so, I think I misunderstood the comment on https://github.com/microsoft/debugpy/pull/61#discussion_r389112538 (which is probably only to don't add a `JUST_MY_CODE` to the `debugOptions` and not remove the `debugOptions` altogether -- just waiting for confirmation to proceed).

**Comment by karthiknadig** [comment]
Dont add `JUST_MY_CODE` to debug options. We have a separate item to deprecate `debug_options` entirely.

**Comment by int19h** [comment]
The tests should be passing both options and flags currently, but once this is done, we should be able to remove all uses of options there.

That said, I don't think it's a big deal if pydevd continues to support them after deprecation. Since we have schema for launch.json, and the editor applies it to validate, the deprecated syntax will be flagged anyway, even if it still works without warning. So pydevd can move on its own schedule, depending on what other direct users are there that could potentially be relying on debug options. 

Basically, from debugpy perspective, the important part for 1.0 is that everything that could be done with debug options, can now be done with individual properties.

**Comment by fabioz** [comment]
OK, I just provided a pull request for that... I also removed the deprecation warnings.
